asus/zephyrus/ga402x/amdgpu/default.nix

```diff
@@ -21,19 +19,6 @@ in {
   };
 
   config = mkMerge [
-    {
-      # AMD RX680
-      services.xserver.videoDrivers = mkDefault [ "amdgpu" ];
-
-      hardware = {
-        amdgpu.loadInInitrd = true;
```

Comment:
The Asus Zephyrus GA402X has a Ryzen CPU, and integrated AMD GPU.

IIRC, I had this enabled because of an issue with needing the AMD GPU enabled earlier in order for the framebuffer console to work correctly before Wayland or X started.

I haven't tried it disabling it since I first enabled it, but assume it's probably still beneficial to keep this on GA402X.

Reply:
Note:  Disabling VPDAU should be possible without also disabling AMD GPU?

Reply:
Apologies, and never mind this comment.
I hadn't noticed the new `../shared.nix` file.

> The Asus Zephyrus GA402X has a Ryzen CPU, and integrated AMD GPU.
> 
> IIRC, I had this enabled because of an issue with needing the AMD GPU enabled earlier in order for the framebuffer console to work correctly before Wayland or X started.
> 
> I haven't tried it disabling it since I first enabled it, but assume it's probably still beneficial to keep this on GA402X.